ecosystem/indexer-grpc/indexer-grpc-data-service/src/service.rs

```diff
@@ -156,14 +166,27 @@ impl RawData for RawDataServerWrapper {
             request_email = request_metadata.request_email.as_str(),
             request_api_key_name = request_metadata.request_api_key_name.as_str(),
             processor_name = request_metadata.processor_name.as_str(),
+            request_connection_id = if let Some(request_connection_id) =
```

Comment:
How come this is an Option? It should always have a connection_id since we created it.

Reply:
done; good point.s